pages/en/indexing/operating-graph-node.mdx

```diff
@@ -0,0 +1,397 @@
+
+```
+[chains]
+ingestor = "block_ingestor_node"
+```
+
+### Supporting multiple networks
+
+The Graph Protocol is increasing the number of networks supported for indexing rewards, and there exist many subgraphs indexing unsupported networks which an indexer would like to process. The `config.toml` file allows for expressive and flexible configuration of:
+- Multiple networks
+- Multiple providers per network (this can allow splitting of load across providers, and can also allow for configuration of full nodes as well as archive nodes, with Graph Node preferring cheaper providers if a given workload allows).
+- Additional provider details, such as features, authentication and the type of provider (for experimental Firehose support)
+
+The `[chains]` section controls the ethereum providers that graph-node connects to, and where blocks and other metadata for each chain are stored. The following example configures two chains, mainnet and kovan, where blocks for mainnet are stored in the vip shard and blocks for kovan are stored in the primary shard. The mainnet chain can use two different providers, whereas kovan only has one provider.
+
+```
+[chains]
+ingestor = "block_ingestor_node"
+[chains.mainnet]
+shard = "vip"
+provider = [
+  { label = "mainnet1", url = "http://..", features = [], headers = { Authorization = "Bearer foo" } },
+  { label = "mainnet2", url = "http://..", features = [ "archive", "traces" ] }
+]
+[chains.kovan]
+shard = "primary"
+provider = [ { label = "kovan", url = "http://..", features = [] } ]
+```
+
+Read more about provider configuration [here](https://github.com/graphprotocol/graph-node/blob/master/docs/config.md#configuring-ethereum-providers).
+
+## Environment variables
+
+Graph Node supports a range of environment variables which can enable features, or change Graph Node behaviour. These are documented [here](https://github.com/graphprotocol/graph-node/blob/master/docs/environment-variables.md).
+
+## Continuous deployment
+
+Users who are operating a scaled indexing setup with advanced configuration may benefit from managing their Graph Nodes with Kubernetes.
+
+> TODO: link to example of Kubernetes setup
+
+> TODO: guidance on monitoring and managing Kubernetes deployments
+
+## Managing Graph Node
+
+Given a running Graph Node (or Graph Nodes!), the challenge is then to manage deployed subgraphs across those nodes. Graph Node surfaces a range of tools to help with managing subgraphs.
+
+### Logging
+
+Graph Node's logs can provide useful information for debugging and optimisation of Graph Node and specific subgraphs. Graph Node supports different log levels via the `GRAPH_LOG` environment variable, with the following levels: debug, error, info, warn, or trace.
+
+> TODO: more detail on logging configuration?
```

Comment:
@lutter are there basics on logging setup or config which we can point people to?

Reply:
As for log management tools? I don't have any good advice, maybe @rotarur has some tips here. Besides setting `GRAPH_LOG`, I also recommend setting `GRAPH_LOG_QUERY_TIMING=gql` to get some details about how GraphQL queries are running, assuming the logging infrastructure can keep up with the amount of logs that produces.

Reply:
thanks good pointer!